### DISCUSSION TITLE
Where is the .fap?

### DISCUSSION THREAD

**Discussion by GitCat3** [dcoles/flipperzero-rs#19] [Q&A] [answered]
First and foremost, I would just like to say you are my lord and savior. However, I am new to rust, and I have a question about where the .fap file is. My cargo workspace is called flipperapp, do I rename the flipperapp binary to flipperapp.fap? Thanks!

**Comment by dcoles** [comment, marked as answer]
Hi @GitCat3,

Glad to hear that this crate has been of use. And yes, that's exactly right - just copy `target/thumbv7em-none-eabihf/{debug,release}/flipperapp` to `flipperapp.rs`. Unfortunately I haven't found an easy way to rename the file automatically.

The [`hello-rust` example](https://github.com/dcoles/flipperzero-rs/tree/main/examples/hello-rust#building) has some instructions showing how to do everything end-to-end.

Happy coding!
